# Contains everything needed to reconstruct a Sample object
time stamp

std_msgs/Float64MultiArray[] sensor_data
